labs-ios-starter/Model/Apollo/Web Auth/Bearer.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Bearer.swift
+//  labs-ios-starter
+//
+//  Created by Kenny on 9/8/20.
+//  Copyright © 2020 Lambda, Inc. All rights reserved.
+//
+
+import Foundation
+
+//this may need to be changed once the backend is deployed
+struct Bearer: Codable {
+    var token: String
+    var username: String
+}
```

Comment:
Do we want to build some persistence into this?

Reply:
Good idea, we could probably use UserDefaults - but it'd be good to check best practices for storing login tokens